### PR TITLE
Draft ietf lpwan schc over lorawan 08

### PR DESCRIPTION
**Pull request by Oliv4945**
- Add uplink All-1 example with last tile
- Fixed IID example
- Use RFC8376 terminology
- List all bitmap possibilities in SCHC ACK example
- Add payload to downlink All-1 example
- Fixed some nits and typos
- Specified retransmition timer
- Clarified downlink opportunity, other frag. parameters
- Described all RFC8724 Annex D

@ivajloip @jca-klk. Would you mind to review it ? Thank you

### CODE REVIEW

**Review by jca-klk — CHANGES_REQUESTED** [review]
Hi,

I tend to disagree with commits 0bb8dbd133b7717a4066b952c1026c9a49415ea7 and 9e300f780c92bc1b27a89922c721513a1cd278d6 : In the context of LoRaWAN, we should keep the LoRaWAN nomenclature. 
IMHO, this document addresses LoRaWAN implementers, we should talk their language.

**Comment by Oliv4945** [comment]
> I tend to disagree 
> IMHO, this document addresses LoRaWAN implementers, we should talk their language.

@jca-klk: Could you please share this concern on the mailing list ? I am not really sure of which choice is the best.

File: draft-ietf-lpwan-schc-over-lorawan.md [hunk 1]
```diff
@@ -763,6 +761,31 @@ inactivity timer is implemented by the device to flush the context in case
...
+### All-1 SCHC fragment
+
+All-1 is distinguishable from a SCHC Sender-Abort as [RFC8724] states *This
+condition is met if the RCS is present and is at least the size of an L2 Word*;
+this condition met: RCS is 4 bytes.
+
+### Delay after each message to respect local regulation
+
+This profile does not define a delay to be added after each SCHC message, local
+regulation compliance will be enforced by LoRaWAN stack.
```
Comment (ivajloip) [inline draft-ietf-lpwan-schc-over-lorawan.md]: Maybe a slightly more precise version would be _regulation compliance is expected to be enforced by LoRaWAN stack._

File: draft-ietf-lpwan-schc-over-lorawan.md [hunk 2]
```diff
@@ -674,92 +688,94 @@ purposes but not SCHC needs.
...
+Class B devices can receive in scheduled RX slots or in RX slots following the
+transmission of an uplink. Class C devices are almost in constant reception.
+For those classes there is no need of an heartbeat to open RX window.
+
+RECOMMENDED retransmission timer value:
+
+* Class B: 3 times the ping slot periodicity.
+* Class C: 30 seconds
+
+The RECOMMENDED transmission timer value is 12 hours for both Class B and Class
```
Comment (arun1587) [inline draft-ietf-lpwan-schc-over-lorawan.md]: this sentence is a bit confusing. is it referring the retransmission timer again?

Comment (arun1587) [reply draft-ietf-lpwan-schc-over-lorawan.md]: I think i see the point now, it is the inactivity timer (perhaps?)  Maybe the context from ln:[725](https://github.com/Acklio/schc-over-lorawan/pull/32/files/5ad91fb7e35b9d9a5ad4427d2535382f9e268f31#diff-f81621167255b169a4bf3c62049366edR725) can be combined and moved out of `Retransmission timer` heading to `Inactivity timer` or something similar.

Comment (Oliv4945) [reply draft-ietf-lpwan-schc-over-lorawan.md]: Thanks Arun, fixed in 139bc6c

File: draft-ietf-lpwan-schc-over-lorawan.md (hunk 1 above)
Comment (Oliv4945) [reply draft-ietf-lpwan-schc-over-lorawan.md]: Thank you, fixed by bcd3993b5b26f05c1425aa8472e35b57a74965db

**Pull request merged**